app/src/main/java/com/keylesspalace/tusky/EditProfileActivity.kt

```diff
@@ -47,7 +47,7 @@ import com.mikepenz.iconics.IconicsDrawable
 import com.mikepenz.iconics.typeface.library.googlematerial.GoogleMaterial
 import com.mikepenz.iconics.utils.colorInt
 import com.mikepenz.iconics.utils.sizeDp
-import com.theartofdev.edmodo.cropper.CropImage
```

Comment:
idk about changes between libraries so don't trust me on checking this :sweat_smile: